### PR TITLE
Embedded video

### PR DESCRIPTION
**Pull request by martynas3113**
- Pop up window on play button click
- Toolbar
- Close on X or "Escape"
- Toolbar shows on mouse move
- Toolbar hides after 5 seconds of mouse inactivity

### CODE REVIEW

**Review by pigma15 — COMMENTED** [review]
in console these 'things' appears

image

File: js/components/VideoPopUp/VideoPopUp.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        })
+        const closeDOM = document.querySelector('.toolbar > span .fa-times')
+        this.closeVideo = closeDOM;
+        this.closeVideo.addEventListener('click', ()=>{
+            this.DOM.classList.add('hidden')
+        })
+        const toolbar = document.querySelector('.pop-up > .toolbar')
+        this.toolBarDOM = toolbar;
+        addEventListener('mousemove',()=>{
+            this.toolBarDOM.classList.remove('hidden')
```
Comment (pigma15) [inline js/components/VideoPopUp/VideoPopUp.js]:
```javascript
addEventListener('mousemove', () => {
    if (this.DOM.classList.contains('hidden')) {
        return // nieko nedaro, kol neatidarytas video langas
    } else {
        if (toolbar.classList.contains('hidden')) {
            toolbar.classList.remove('hidden'); // parodo ikonas vos tik atsidarius video langui ir sujudejus cursoriui
        } else {
            const timer = // jei ikonos matomos inicijuoja timeri, kad jos dingtu uz 5 sekundziu
                setTimeout(() => {
                    toolbar.classList.add('hidden')
                }, 5000)
            addEventListener('mousemove', ()=> { // jei ikonos matomos, bet pele vis dar juda timeris cancel'inamas ir inicijuojamas is naujo del auksciau buvusio eventListenerio
                if (!toolbar.classList.contains('hidden')) {
                    clearTimeout(timer);
                }
            })
        }
    }
})
```

**Comment by martynas3113** [comment]
> in console these 'things' appears
> 
> image

Cia sitie atsiranda del to video ikelto. Kazkas su Google ads susije. Bet tai nedaro itakos veikimui.

**Review by pigma15 — APPROVED** [review]
lgtm

**Review by bemze — APPROVED** [review]
LGTM

**Pull request merged**